base_tier_validation/static/src/components/tier_review_menu/tier_review_menu.xml

```diff
@@ -0,0 +1,71 @@
+<t t-name="base_tier_validation.TierReviewMenu">
+        <Dropdown
+            position="'bottom-end'"
+            state="dropdown"
+            beforeOpen.bind="onBeforeOpen"
+            menuClass="discussSystray.menuClass"
+        >
+            <button>
+                <i
+                    class="fa fa-lg fa-pencil-square-o"
+                    role="img"
+                    aria-label="Reviews"
+                />
+                <span
+                    t-if="state.tierReviewCounter > 0"
+                    class="o-mail-ActivityMenu-counter badge rounded-pill"
+                    t-out="state.tierReviewCounter"
+                />
+            </button>
+            <t t-set-slot="default">
+                <div t-att-class="`${discussSystray.contentClass} o-mail-ActivityMenu`">
+                    <DropdownItem>
+                        <div
+                            t-if="state.tierReviewCounter === 0"
+                            class="o-mail-ActivityMenu-empty align-items-center opacity-50 d-flex justify-content-center"
+                        >
+                            <span>No reviews to do.</span>
+                        </div>
+                    </DropdownItem>
+                    <div
+                        class="d-flex flex-column list-group-flush"
+                        name="activityGroups"
+                    >
+                        <t
+                            t-foreach="state.tierReviewGroups"
+                            t-as="group"
+                            t-key="group_index"
+                            name="activityGroupLoop"
+                        >
+                            <div
+                                class="o-mail-ActivityGroup list-group-item list-group-item-action d-flex cursor-pointer"
+                                t-att-data-model_name="group.model"
+                                t-on-click="() => this.openReviewGroup(group)"
+                            >
+                                <img alt="Activity" t-att-src="group.icon" />
+                                <div class="flex-grow-1 overflow-hidden">
+                                    <div
+                                        class="d-flex px-2"
+                                        name="activityTitle"
+                                        t-out="group.name"
+                                    />
+                                    <div class="d-flex">
+                                        <span
+                                            t-attf-class="#{group.pending_count ? '' : 'text-muted'} py-0 px-2"
+                                        >
+                                        <t t-out="group.pending_count" /> Pending
+                                    </span>
+                                    </div>
+                                </div>
+                            </div>
+                        </t>
+                    </div>
+                </div>
```

Comment:
I tested  with this PR https://github.com/OCA/sale-workflow/pull/3417
The UI is broken; please fix it.
Without revisions, an icon must be displayed instead of text.
V18
![image](https://github.com/user-attachments/assets/37ea700a-91e0-4f04-89ef-a018b6c83e2d)
V17
![image](https://github.com/user-attachments/assets/bde5e3b9-f501-4b2d-9bd0-ebafdd80befb)

The revision count is hidden, and the systray is broken.

V18
![image](https://github.com/user-attachments/assets/468f26e2-5e2a-436c-8db8-52c0af14eeab)

V17
![image](https://github.com/user-attachments/assets/6115ee6a-9d8a-479c-b885-04b969814928)